Description: don't require wast-loader which depends on wabt, not yet packaged
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2021-10-17

--- a/test/setupTestFramework.js
+++ b/test/setupTestFramework.js
@@ -130,6 +130,6 @@
 // Workaround for a memory leak in wabt
 // It leaks an Error object on construction
 // so it leaks the whole stack trace
-require("wast-loader");
+//require("wast-loader");
 process.removeAllListeners("uncaughtException");
 process.removeAllListeners("unhandledRejection");
